index.php

```diff
@@ -153,6 +156,9 @@
 	case "mysqli":
 		$db = new DB_MySQLi;
 		break;
+	case "mysql_pdo":
+		$db = new MysqlPdoDbDriver;
+		break;
```

Comment:
There's another `PostgresPdoDbDriver` class, you may also want to add it. By the way, it might be better to append brackets to the classnames here as done in MyBB's `inc/init.php`

https://github.com/mybb/mybb/blob/89ba6fd39fca4ede328f69deca9aba7b57f2c5d1/inc/init.php#L130-L149